app/views/training/pages/_timestamp.html.slim

```diff
@@ -0,0 +1,8 @@
+- if !content.published_at.nil?
```

Comment:
`content.parent.published?` would be better. Make sure that the timestamps are still accurate when the module was already published and an editor is injecting a brand new page.